--- serf-1.2.1/configure.in.orig	2013-06-06 15:24:17.713217048 +0200
+++ serf-1.2.1/configure.in	2013-06-06 15:33:22.779872353 +0200
@@ -182,10 +182,8 @@
     APR_ADDTO(CPPFLAGS, "-I${openssl_prefix}/include")
     if test -e "${openssl_prefix}/Makefile"; then
       APR_ADDTO(LDFLAGS, "-L${openssl_prefix}")
-      APR_ADDTO(LDFLAGS, "-R${openssl_prefix}")
     else
       APR_ADDTO(LDFLAGS, "-L${openssl_prefix}/lib")
-      APR_ADDTO(LDFLAGS, "-R${openssl_prefix}/lib")
     fi
   fi
 ])
